<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetNetwork</class>
 <widget class="QWidget" name="WidgetNetwork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>421</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>652</width>
    <height>387</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>网络调试助手</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>471</width>
       <height>231</height>
      </size>
     </property>
     <property name="title">
      <string>接收窗口</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" rowspan="4">
       <widget class="QTextEdit" name="textEdit_recive">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_ReciveHex">
        <property name="text">
         <string>十六进制显示</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>141</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_ReciveSave">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_ReciveClean">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>清除</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>235</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>网络设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_Protocol">
        <property name="text">
         <string>协议选择</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="comboBox_Protocol">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>tcp server</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>tcp client</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>udp</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_IPAddr">
        <property name="text">
         <string>本地IP地址</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineEdit_IPAddr">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_Port">
        <property name="text">
         <string>本地端口</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="lineEdit_Port">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pushButton_Connect">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_ConnectObject">
        <property name="text">
         <string>通讯对象</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QComboBox" name="comboBox_ConnectObject">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>ALL</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_1">
     <property name="minimumSize">
      <size>
       <width>601</width>
       <height>141</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>141</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>发送窗口</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" rowspan="2">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true">#stackedWidget{
background-color: rgba(229, 229, 229, 0);
}</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_3">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" rowspan="4">
           <widget class="QTextEdit" name="textEdit_send">
            <property name="minimumSize">
             <size>
              <width>481</width>
              <height>87</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>81</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QCheckBox" name="checkBox_SendHex">
            <property name="maximumSize">
             <size>
              <width>91</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>十六进制发送</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBox_Timer">
            <property name="maximumSize">
             <size>
              <width>42</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>定时</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEdit_Timer">
            <property name="maximumSize">
             <size>
              <width>31</width>
              <height>14</height>
             </size>
            </property>
            <property name="text">
             <string>1000</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_Timer">
            <property name="maximumSize">
             <size>
              <width>41</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>ms</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QPushButton" name="pushButton_Send">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>91</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QPushButton" name="pushButton_SeendClean">
            <property name="minimumSize">
             <size>
              <width>31</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>91</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>清除</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_4">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLineEdit" name="lineEdit_shortcut1"/>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_shortcut1">
              <property name="minimumSize">
               <size>
                <width>3</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>发送</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLineEdit" name="lineEdit_shortcut2"/>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_shortcut2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>发送</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="lineEdit_shortcut3"/>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_shortcut3">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>发送</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLineEdit" name="lineEdit_shortcut4"/>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="pushButton_shortcut4">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>发送</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButton">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>18</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>18</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>82</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
